python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -365,22 +365,24 @@ def __get__(self, obj, cls):
             return cls._schema[name]
 
         if self._getter_name:
-            return self._bind_method(obj, self._getter_name)()
+            returnable = self._bind_method(obj, self._getter_name)()
+            if isinstance(returnable, GangaObject):
+                returnable._setParent(self)
+            return returnable
```

Comment:
Given this bug is causing infinite loops and hanging when submitting even simple test jobs I'd like to push out 6.1.16 ASAP, maybe even today. It's causing python/Ganga/new_tests/GPI/TestArgSplitter.py to hang forever (reproducible on the interactive prompt).